### PR TITLE
Feature/1 add recommendations

### PR DESCRIPTION
**Pull request by WhiT3Kr0w**
I've adapted code to the new redux store. Had to make a few changes and also simplified it a little bit.

Still working on a few fixes, but recommendations are being stored in mongoDB and we are able to fetch results from the DB as well (using contributor '099' as a temp placeholder until we have auth setup).

### CODE REVIEW

File: api/v1/recommendations/index.js [hunk 1]
```diff
@@ -1,8 +1,36 @@
...
+import { addRecommendation, getDocuments } from '../../../connect/db';
 
 const getCollectionResults = async (req, res) => {
-  const documents = await getDocuments('recommendations');
-  return res.status(200).json({ documents });
+  const { cid } = req.query;
+
+  let result;
+
+  if (cid) {
```
Comment (siims) [inline api/v1/recommendations/index.js]: not clear what is happening here. if-else where branches compare different things is very strange to read.
I assume that intention is that top level if layer differentiates based on http method.

File: api/v1/recommendations/index.js [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
+      },
+      {
+        food_id: 1,
+        recommendation_id: 1,
+        contributor_id: 1,
+        _id: 0,
+      }
+    );
+  } else if (req.method === 'POST') {
+    result = [req.body[0]];
```
Comment (siims) [inline api/v1/recommendations/index.js]: it is not clear why first element of the body array is handled differently than others.

Comment (WhiT3Kr0w) [reply api/v1/recommendations/index.js]: This should be an empty array. I've populated it and added a console.log to test and forgot to remove it.
I've changed it back.

File: api/v1/recommendations/index.js (hunk 1 above)
Comment (WhiT3Kr0w) [reply api/v1/recommendations/index.js]: Yes. If the query string contains a cid parameter it does one thing, if it doesn't, then it checks for a POST request.

**Pull request merged**